ambry-store/src/main/java/com/github/ambry/store/LogSegment.java

```diff
@@ -532,10 +551,12 @@ void close(boolean skipDiskFlush) throws IOException {
     if (open.compareAndSet(true, false)) {
       if (!skipDiskFlush) {
         flush();
+        directIOExecutor.flush();
```

Comment:
Isn't this redundant? 
We have added the directIOExecutor.flush inside the flush() already.

Reply:
You are right.